### DISCUSSION TITLE
collaboration on 2D flow tube chemistry ("flotus")

### DISCUSSION THREAD

**Discussion by ngam** [uncscode/particula#20] [Ideas]
Lance (@momo-catcat) and Jiali are working on a 2D flow-tube chemistry module in the coming months and would like to eventually combine it with Neil's code (which we will be translating). I am opening this discussion topic to document the interest and to keep this in mind as we add modules in the coming months. We will be starting slowly on various topics (pertinent to our individual expertise) and then we will move to refactoring and reconciling code as we go.

**Comment by ngam** [comment]
@momo-catcat, could you tag Jiali and other interested people? Maybe @Mingyi-W is also interested 👀

**Reply by momo-catcat** [reply]
Here she is:

@Jialishen
